cypress/support/custom_commands/loginLogout.js

```diff
@@ -26,7 +26,7 @@ Cypress.Commands.add('login', (username, environment) => {
     let user_password
```

Comment:
File name `loginLogout.js` should be replaced by `authentication.js` or something similar. Frankly speaking its not look good.

Reply:
- currently in main file name is called -> login.js , but it also contains 'logout' custom command in it, 
- so if we keep file name as authentication.js  -> it implies only login because then 'logout' is 'unauthenticating' process, should we keep then 'logout' custom command in a separate file called -> logout.js ?

Reply:
I would say we can adapt another meaningful name in a follow up, because the file name 'loginLogout.js' is the same as in the main at the moment.

Reply:
updated and created two files, named 'login.js' and 'logout.js'.